driver/pelican/interface.md

```diff
@@ -0,0 +1,40 @@
+### Pelican Schedule Interface Outline
+
+##### Context
+
+The following structs define the way users interact with the Scheduling interface of the Pelican thermostats. These structs are used to define the weekly schedule and will be interpretted by the thermSchedule.go code to retrieve/view in addition to making changes to the existing schedule for individual thermostats within a particular site. These structs are public and accessible to anyone who subscribes to the allotted endpoint or publishes to the assigned signal.
+
+##### Schedule Structs
+
+// Struct mapping each day of the week to its daily schedule <br>
+type ThermostatSchedule struct {<br>
+&nbsp;&nbsp;&nbsp;DaySchedules map[string]ThermostatDaySchedule `msgpack:"day_schedules"`<br>
```

Comment:
Where is the time of each schedule epoch captured? It would be good to document what the string keys of the `DaySchedules` attribute are

Reply:
No problem, I added an explanation for each of the fields within ThermostatBlockSchedule in a separate section. I also explain how the time + RRule format works!